src/libs/PaymentUtils.js

```diff
@@ -29,11 +30,31 @@ function hasExpensifyPaymentMethod(cardList = [], bankAccountList = []) {
  * @param {Array} bankAccountList
  * @param {Array} cardList
  * @param {Object} [payPalMeData = null]
+ * @param {Object} pendingBankAccount
  * @returns {Array<PaymentMethod>}
  */
-function formatPaymentMethods(bankAccountList, cardList, payPalMeData = null) {
+function formatPaymentMethods(bankAccountList, cardList, payPalMeData = null, pendingBankAccount = {}) {
     const combinedPaymentMethods = [];
 
+    // See if we need to show a pending bank account in the payment methods list
+    if (!_.isEmpty(pendingBankAccount)) {
```

Comment:
Is this data massaging something we can do in OpenPaymentsPage instead?

Reply:
I don't think so. This pending bank account is all local data. None of this is stored on the server-side. Just like plaid data

Reply:
Is this data massaging? Something can be empty or undefined.

Reply:
> Is this data massaging?

Ah, sorry, I wasn't clear. I meant the data massaging on lines 43 - 54.

But if this data is all local data then I don't see how we can avoid this 🤔